modules/contacts/client/components/ContactPresentational.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withNamespaces } from 'react-i18next';
+import Avatar from '@/modules/users/client/components/Avatar.component';
+
+export function ContactPresentational({ t, contact, avatarSize=128, hideMeta=false, situation, onClickRemove }) {
+  return (<>
+    <Avatar user={contact.user} size={avatarSize} />
+    <h4>
+      <a href={`/profile/${contact.user.username}`}>{contact.user.displayName}</a>
+    </h4>
+    {contact.user.locationLiving && <div>
+      <i className="icon-fw icon-building text-muted"></i>
+      <small>
+        {t('Lives in')}
+        <a href={`/search?location=${contact.user.locationLiving}`}>{contact.user.locationLiving}</a>
+      </small>
+    </div>}
+    {contact.user.locationFrom && <div>
+      <i className="icon-fw icon-home text-muted"></i>
+      <small>
+        {t('From')}
```

Comment:
:+1: 

`t` supports parameters via
```js
t('some string with {{parameter, optionalFormat}}', { parameter: 'city1' })
```
And there is also `react-i18next`'s [`<Trans>`](https://react.i18next.com/components/trans-component) component which simplifies dealing with text that contains html.

This will be supported out of the box. I was just lazy to re-research. :snail: :bear: :cat2: Will fix.

Reply:
We'll have plenty of these cases so would be great to get an example for this in the codebase so that everyone can learn and copy from it. :-)

Reply:
15f8cba5a99783deab9b254792a76a43bb56f99b